### PR TITLE
Gh 2542

### PR DESCRIPTION
**Pull request by PCF-Testing**
Separate permissions for SU, staff, and partner level users.

### CODE REVIEW

File: apps/profiles/templatetags/profiles_tags.py [hunk 1]
```diff
@@ -72,6 +72,19 @@ def _user_needs_languages(context):
...
+
+    if user.is_authenticated():
+        context['users_actions'] = Action.objects.select_related('video', 'language', 'language__video', 'user') \
+            .filter(video__customuser=user) \
+            .exclude(user=user) \
+            .exclude(user=User.get_amara_anonymous())[:ACTIONS_ON_PAGE]
+    else:
+        context['users_actions'] = Action.objects.none()
+    return context
+
```
Comment (bendk) [inline apps/profiles/templatetags/profiles_tags.py]: This seem unrelated to the rest of the changes.  Did it come from a merge mistake?

**Comment by bendk** [comment]
Looks great, just wondering if one of the changes came from a bad merge.  Once we figure that out, I'm happy to merge.

**Comment by syl22-00** [comment]
You are correct, it's a function I modified while you removed it as it was not used anymore. While resolving the conflict, I, wrongly, chose my version. Just pushed the fix.
